src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf

```diff
@@ -402,6 +402,30 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Tīkla maskas (netmask) vērtībai jābūt starp {{ min }} un {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Faila nosaukums ir pārāk garš. Tas var būt { filename_max_length }} rakstzīmi vai īsāks.|Faila nosaukums ir pārāk garš. Tajā var būt {{ filename_max_length }} rakstzīmes vai īsāks.</target>
```

Comment:
missing {  
also "rakstzīme" not "rakstzīmi"  
also for consistency - "Tajā var būt" should be replaced by "Tas var būt"